fili-core/src/test/groovy/com/yahoo/bard/webservice/table/ConcretePhysicalTableSpec.groovy

```diff
@@ -86,7 +86,20 @@ class ConcretePhysicalTableSpec extends Specification {
         dimensionColumn             | intervalSet1
         metricColumn1               | intervalSet2
         metricColumn2               | intervalSet3
-        new Column("MissingName")   | [] as Set
+    }
+
+    @Unroll
```

Comment:
No need for `@Unroll` here, since there's no `where` block and no templated test name.